pages/api/senado/v1/quantitativos/cargos-funcoes.js

```diff
@@ -1,20 +1,13 @@
 import app from '@/app';
-import BadRequestError from '@/errors/BadRequestError';
-import NotFoundError from '@/errors/NotFoundError';
+import InternalError from '@/errors/InternalError';
 import { getDadosQuantitativos } from '@/services/dados-abertos-br/senado-quantitativos';
 
 async function getDadosSenadoQuantitativo(request, response) {
   try {
     const result = await getDadosQuantitativos();
     return response.status(result.status).json(result.data);
```

Comment:
Como a API deles podem retornar qualquer status code [200, 299] recomendo que forcemos um 200. Deste modo se eles não estiverem usando o status code correto, nossos usuários não serão afetados.

```suggestion
    return response.status(200).json(result.data);
```